packages/components/src/components/domainproperties/issues/IssuesPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,221 @@
+    model: IssuesModel;
+    coreGroups?: List<Principal>;
+    onSelect?: (selected: Principal) => any;
+    coreUsers?: List<UserGroup>;
+}
+
+export class IssuesBasicPropertiesFields extends React.PureComponent<IssuesBasicPropertiesInputsProps> {
+    render() {
+        const { model, onInputChange } = this.props;
+        return (
+            <Col xs={11} md={6}>
+                <SectionHeading title="Basic Properties" />
+                <CommentSortDirectionDropDown model={model} onInputChange={onInputChange} />
+                <SingularItemNameInput model={model} onInputChange={onInputChange} />
+                <PluralItemNameInput model={model} onInputChange={onInputChange} />
+            </Col>
+        );
+    }
+}
+
+export class AssignmentOptions extends React.PureComponent<SecurityUserGroupProps> {
+    render() {
+        const { model, coreUsers, coreGroups, onSelect } = this.props;
+        return (
+            <Col xs={11} md={6}>
+                <SectionHeading title="Assignment Options" />
+                <AssignedToGroupInput
+                    model={model}
+                    coreGroups={coreGroups}
+                    onSelect={(selected: Principal) => onSelect(selected)}
+                />
+                <DefaultUserAssignmentInput model={model} coreUsers={coreUsers} onSelect={(selected: Principal) => onSelect(selected)} />
+            </Col>
+        );
+    }
+}
+
+export class SingularItemNameInput extends React.PureComponent<IssuesBasicPropertiesInputsProps> {
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.name === null ? '' : model.name;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Singular item name" required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        id="singularItemName"
+                        type="text"
+                        placeholder="Enter a singular name for this Issue"
+                        value={value}
+                        onChange={onInputChange}
+                    />
+                </Col>
+
+                <Col lg={2} />
+            </Row>
+        );
+    }
+}
+
+export class PluralItemNameInput extends React.PureComponent<IssuesBasicPropertiesInputsProps> {
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.name === null ? '' : model.name;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Plural items name" required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        id="pluralItemName"
+                        type="text"
+                        placeholder="Enter a plural name for this Issue"
```

Comment:
I think we should remove this placeholder text and instead add in a help tooltip text to the DomainFieldLabel. I can help come up with the text for that tooltip.

Reply:
done